src/ui/views/Dashboard/index.tsx

```diff
@@ -683,22 +685,32 @@ const Dashboard = () => {
     setTopAnimate('fadeInTop');
   };
   const removeToken = async (removeToken) => {
-    const newAddTokenList = addedToken.filter(
-      (item) => item !== removeToken?.id
+    const uuid = `${removeToken?.chain}:${removeToken?.id}`;
```

Comment:
variable `removeToken` use the same name with function `removeToken`, how about changing the variable to `token`?

Reply:
done